### DISCUSSION TITLE
Remote access to Raspberry

### DISCUSSION THREAD

**Discussion by merlinste** [zivillian/ism7mqtt#143] [Q&A]
Hey all,

First of all, I am not an Raspberry expert by any means :) 
I am running ism7mqtt at home and am quite happy with it.
Now I have to take care of 4 CHA16 in a bigger multi-purpose building. First I tried to scrape the data via the „Wolf Smart“ HA-Addon, but most of the interesting data is not being delivered.
Is there any way I can install a Raspberry in the building and „stream“ the data to my home-raspberry and integrate the data of the 4 16ers?
Or is a complete setup of a new Raspberry and access via VPN the best idea?

Thanks for any ideas!

**Comment by Eba-M** [comment]
You can run one or more instance for each CHA16 using screen. Within screen runs ism7mqtt for each CHA16
All connect to the local mqtt broker
you can connect to the raspberry via ssh.
